src/chrome/content/rules/148apps.com.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Nonfunctional hosts in *.148apps.com:
+	    beta.images.148apps.com
+	Redirect to "www.148apps.com":
+	    ads.148apps.com
+	    forum.148apps.com
+	    images.148apps.com
+    	    img.148apps.com
+	    media.148apps.com
+    	    beta.media.148apps.com
+	    static.148apps.com
+	    beta.static.148apps.com
+            store.148apps.com
+
+	Http redirect:
+	    www.148apps.com
+-->
+<ruleset name="148apps">
```

Comment:
The site has horrible mixed content issues, so please add the mixedcontent platform.